config/maps.txt

```diff
@@ -41,6 +41,14 @@ map deltastation
 	votable
 endmap
 
+map deadstation
+	minplayers 0
+	maxplayers 25
+	votable
+endmap
+
+#Does not support fastmos
```

Comment:
Any point for this message?

Reply:
Honestly, unless a fastmos PR comes along there isn't any point to that. If you want me to remove it I can.

Reply:
Yea, probably best since all our maps use the new firelocks if that's what you mean by does not support fastmos

Reply:
removed in 5aed385